releases/web-helm-release.yaml

```diff
@@ -50,8 +50,7 @@ spec:
       env:
       - name: PORT
         value: "4200"
-      # - name: SSO_URL
-      #   value: "https://keycloak.global.stakater.com"
+
```

Comment:
why this file is required as its replicate exists here `releases-istio/apps/web-helm-release.yaml`?

Reply:
Both are different directories, flux will watch either of those based on if user wants nordmart with istio or without it